set.mm

```diff
@@ -53537,6 +53554,29 @@ of the relation (see ~ op1stb ).  (Contributed by NM, 28-Jul-2004.) $)
       NRABGOABCDEHFPQ $.
   $}
 
+  ${
+    $d D x $.  $d x A $.  $d x C $.  $d x ph $.
+    elrnmptspd.1 $e |- F = ( x e. A |-> B ) $.
+    elrnmptspd.2 $e |- ( ph -> C e. A ) $.
+    elrnmptspd.3 $e |- ( ph -> D e. V ) $.
+    elrnmptspd.4 $e |- ( ( ph /\ x = C ) -> D = B ) $.
+    $( Elementhood in an image set.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    elrnmptspd $p |- ( ph -> D e. ran F ) $=
+      ( crn wcel wceq wrex rspced wb elrnmpt syl mpbird ) AFGMNZFDOZBCPZAUCBECL
+      JQAFHNUBUDRKBCDFGHISTUA $.
+  $}
+
+  ${
+    $d x C $.
+    elrnmpt2d.1 $e |- F = ( x e. A |-> B ) $.
+    elrnmpt2d.2 $e |- ( ph -> C e. ran F ) $.
+    $( Elementhood in an image set.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
```

Comment:
Comment to be revised, see above.

Reply:
Used the same comment as in ` elrnmptdv `.